### ISSUE TITLE
Add LICENSE file

### ISSUE THREAD

**Issue by danny-laa** [ProtonMail/proton-calendar#4]
If this project was meant to be FOSS, please add a LICENSE file, so that GitHub can detect and display license informations.

**Comment by arhayati** [comment]
Yes, please. GNU GPLv3+ or even better, GNU AGPLv3+ would be awesome.

**Comment by WarenGonzaga** [comment]
Looks like this issue can be close now.
